src/test/java/uk/gov/companieshouse/certificates/orders/api/controller/CertificateItemsControllerIntegrationTest.java

```diff
@@ -1798,6 +1826,242 @@ void updateCertificateItemRejectsConflictingDetailsSettings() throws Exception {
                 .andDo(MockMvcResultHandlers.print());
     }
 
+    @Test
+    void correctlyErrorsWhenActiveLimitedCompanyAndLiquidatorsDetailsSupplied()
```

Comment:
Parameterise test using ParameterizedTest annotation.

Reply:
All six liquidators details tests in this suite are now performed by a single test method. As advised parameterisation has been implemented using the PararameterizedTest annotation. Test parameters are now provided to the test method by a new CertificateItemsFixture [builder] class.